<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClustalWMsaBuilderOptionsWidget</class>
 <widget class="IMsaBuilderOptionsWidget" name="ClustalWMsaBuilderOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Output Order:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="orderByAlignedRadioButton_">
        <property name="text">
         <string>Aligned</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="orderByInputSequencesRadioButton_">
        <property name="text">
         <string>Input sequences</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Tree Parameters:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Guide tree:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="guideByNJRadioButton_">
        <property name="text">
         <string>Neighbor joining</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="guideByUPGMARadioButton_">
        <property name="text">
         <string>UPGMA</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="excludeGapPositionsCheckBox_">
        <property name="text">
         <string>Exclude positions with gaps</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QCheckBox" name="correctMultipleSubstitutionsCheckBox_">
        <property name="text">
         <string>Correct for multiple substitutions (Kimura correction)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Pairwise Alignments:</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="slowPairwiseAlignmentsRadioButton_">
        <property name="text">
         <string>Slow</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fastPairwiseAlignmentsRadioButtion_">
        <property name="text">
         <string>Fast</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="editPairwiseParametersButton_">
        <property name="text">
         <string>Edit Parameters...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Multiple Alignment:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gap open penalty:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="AgDoubleSpinBox" name="gapOpenPenaltyDoubleSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gap extension penalty:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="AgDoubleSpinBox" name="gapExtensionPenaltyDoubleSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Delay divergent sequences:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QSpinBox" name="delayDivergentSequencesSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Weight matrix:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QComboBox" name="weightMatrixComboBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QCheckBox" name="disableSequenceWeightingCheckBox_">
        <property name="text">
         <string>Disable sequence weighting</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="iterateCheckBox_">
        <property name="text">
         <string>Iterate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="iterateByAlignmentRadioButton_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Alignment</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QRadioButton" name="iterateByTreeRadioButton_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Tree</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4" colspan="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>135</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Number of iterations:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="3" colspan="2">
       <widget class="QSpinBox" name="numIterationsSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IMsaBuilderOptionsWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/IMsaBuilderOptionsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AgDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui/widgets/AgDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>iterateCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>iterateByAlignmentRadioButton_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterateCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>iterateByTreeRadioButton_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterateCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterateCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>numIterationsSpinBox_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapOpenPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapExtensionPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>delayDivergentSequencesSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>weightMatrixComboBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
